spec/features/operator/curatedtasks/clustertask_test.go

```diff
@@ -0,0 +1,18 @@
+package curatedtasks
+
+import (
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestClusterTasksValidation(t *testing.T) {
+	Convey("Given the Operator is installed", t, func() {
+		Convey("When I logged in as non-admin user", func() {
+			Convey("Then I should be able to validate ClusterTask manifest yamls ", nil)
```

Comment:
By `validate` do we mean "view" or do we mean "create a ClusterTask1 ?

Reply:
or create a `TaskRun` using that `ClusterTask` and having it work ?

Reply:
it's lint tests actually, TaskRun, PipelineRun using ClusterTask is already defined in another test